NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -50,32 +72,101 @@ def begin(
         ----------
         debug: bool, default: False
             It True, debug plots will be shown
+        skymodel: {'lfmap', 'lfss', 'gsm2016', 'haslam', 'ssm', 'gmoss', 'ulsa_fdi', 'ulsa_dpi', 'ulsa_ci'}, optional
+            Choose the sky model to use. If none is provided, the Global Sky Model (2008) is used as a default.
         n_side: int, default: 4
             The n_side parameter of the healpix map. Has to be power of 2
             The radio skymap is downsized to the resolution specified by the n_side
             parameter and for every pixel above the horizon the radio noise coming
             from that direction is calculated. The number of pixels used is
             12 * n_side ** 2, so a larger value for n_side will result better accuracy
             but also greatly increase computing time.
-        interpolation_frequencies: array of float
+        freq_range: array of len=2, default: [30, 80] * units.MHZ
             The sky brightness temperature will be evaluated for the frequencies
-            in this list. Brightness temperature for frequencies in between are
+            within this limit. Brightness temperature for frequencies in between are
             calculated by interpolation the log10 of the temperature
             The interpolation_frequencies have to cover the entire passband
             specified in the run method.
         """
         self.__debug = debug
         self.__n_side = n_side
-        self.__interpolaiton_frequencies = interpolation_frequencies
+
+        if freq_range is None:
+            freq_range = np.array([30, 80]) * units.MHz
```

Comment:
Relatedly - changing from a default of [10,1100] to [30,80] MHz is quite a big change, and e.g. for RNO-G the new default is definitely not what we want. I would prefer to keep the old default and just raise an error (or a warning) if the chosen model is not compatible with the frequency range.

Reply:
(oops, I changed the default for LOFAR related reasons but meant to change it back for the commit)

Reply:
spppppppace :)